drivers/Makefile.dep

```diff
@@ -18,6 +18,11 @@ ifneq (,$(filter at24%,$(USEMODULE)))
   endif
   ifneq (,$(filter at24mac,$(USEMODULE)))
     USEMODULE += at24xxxx
+# To keep functionality the same for external boards we are adding the
+# at24mac_id, even though it can be use independently.
+# This should be removed as of 2022.07 release.
+    USEMODULE += at24mac_id
+    DEPRECATED_MODULES += at24mac
```

Comment:
If `at24mac` is deprecated, how do I select the `at24mac` EEPROM?

Reply:
Wow you are fast, before I can even finish my comment :)

Reply:
I would have liked to add a custom note but then we see how recursive the build system is...

Reply:
My hope is that this would be enough to get people to look into this deprecation note in the docs to see that it isn't removal of eeprom just the id.

Reply:
I am super open to better ways though...